src/tactic/rename_var.lean

```diff
@@ -0,0 +1,45 @@
+import tactic.interactive
```

Comment:
Missing header? (I am not competent on tactic writing, but I can bikeshed on trivial issues :)

Reply:
Good catch. There is no reason tactic files could escape the module docstring requirements.